yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -561,45 +567,92 @@ func getAssessmentReportContentFromAnalyzeSchema() (err error) {
 	}
-	assessmentReport.UnsupportedFeatures = unsupportedFeatures
-
+	assessmentReport.UnsupportedFeatures = append(assessmentReport.UnsupportedFeatures, unsupportedFeatures...)
 	return nil
 }
 
-func fetchUnsupportedFeaturesOfSourceInYb(schemaAnalysisReport utils.SchemaReport) ([]UnsupportedFeature, error) {
-	switch source.DBType {
-	case POSTGRESQL:
-		return fetchUnsupportedFeaturesForPG(schemaAnalysisReport)
-	case ORACLE:
-		return nil, nil
-	default:
-		panic("unsupported source db type")
+var filterIssues = func(featureName, issueReason string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature) {
+	log.Info("filtering issues for feature: ", featureName)
+	objectNames := make([]string, 0)
+	for _, issue := range schemaAnalysisReport.Issues {
+		if strings.Contains(issue.Reason, issueReason) {
+			objectNames = append(objectNames, issue.ObjectName)
+		}
 	}
+	*unsupportedFeatures = append(*unsupportedFeatures, UnsupportedFeature{featureName, objectNames})
 }
 
-func fetchUnsupportedFeaturesForPG(schemaAnalysisReport utils.SchemaReport) ([]UnsupportedFeature, error) {
+func fetchUnsupportedPGFeaturesFromSchemaReport(schemaAnalysisReport utils.SchemaReport) ([]UnsupportedFeature, error) {
 	log.Infof("fetching unsupported features for PG...")
 	unsupportedFeatures := make([]UnsupportedFeature, 0)
-	filterIssues := func(featureName, issueReason string) {
-		log.Info("filtering issues for feature: ", featureName)
-		objectNames := make([]string, 0)
-		for _, issue := range schemaAnalysisReport.Issues {
-			if strings.Contains(issue.Reason, issueReason) {
-				objectNames = append(objectNames, issue.ObjectName)
-			}
-		}
-		unsupportedFeatures = append(unsupportedFeatures, UnsupportedFeature{featureName, objectNames})
+	filterIssues("GIST indexes", GIST_INDEX_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	filterIssues("Constraint triggers", CONSTRAINT_TRIGGER_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	filterIssues("Inherited tables", INHERITANCE_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	filterIssues("Tables with Stored generated columns", STORED_GENERATED_COLUMN_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	return unsupportedFeatures, nil
+}
+
+func fetchUnsupportedOracleFeaturesFromSchemaReport(schemaAnalysisReport utils.SchemaReport) ([]UnsupportedFeature, error) {
+	log.Infof("fetching unsupported features for Oracle...")
+	unsupportedFeatures := make([]UnsupportedFeature, 0)
+	filterIssues("Compound Triggers", COMPOUND_TRIGGER_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	return unsupportedFeatures, nil
+}
+
+var OracleUnsupportedIndexTypes = []string{"CLUSTER INDEX", "DOMAIN INDEX", "BITMAP INDEX", "FUNCTION-BASED DOMAIN INDEX", "IOT - TOP INDEX", "NORMAL/REV INDEX", "FUNCTION-BASED NORMAL/REV INDEX"}
+
+func fetchUnsupportedObjectTypes() ([]UnsupportedFeature, error) {
+	if source.DBType != ORACLE {
```

Comment:
In pg also we have multiple types of indexes(INET) and inherited tables, etc that are not supported right? Where are those caught? I would have expected them also to be here itself.